scripts/ci/source_code_static_analysis.py

```diff
@@ -86,7 +86,7 @@ def _run_pylint(module_paths, ignored_modules=None, rcfile=None, cpu_count=1):
         # 8:  Refactor message issued
         # 16: Convention message issued
         # 32: Usage error
-        if se.code not in [0, 8, 16, 24]:
+        if se.code != 0:
```

Comment:
Why do you change this logic?

Reply:
I miss out one thing, it's that wrong-import-order (which in _scripts/ci/souce_code_static_analysis.py_ ) is also a convention level suggestion which will fail old script _**scripts/ci/test_static.sh** but my new-added script don't.

So, I let them open, along with the criteria with old scripts.